src/touch/touch.rs

```diff
@@ -52,7 +55,7 @@ macro_rules! to_timeval {
 }
 
 #[cfg(unix)]
-fn set_symlink_times(p: &str, atime: FileTime, mtime: FileTime) -> io::Result<()> {
+fn set_symlink_file_times(p: &str, atime: FileTime, mtime: FileTime) -> io::Result<()> {
```

Comment:
I don't think this function as actually necessary even on Unix.  `set_symlink_file_times()` in `filetime` seems to say it does the same thing.

Reply:
🛠 * both fixed